macosx/CreatorWindowController.mm

```diff
@@ -617,88 +620,24 @@ - (void)createReal
     self.window.restorable = NO;
 
     [NSNotificationCenter.defaultCenter postNotificationName:@"BeginCreateTorrentFile" object:self.fLocation userInfo:nil];
-    tr_makeMetaInfo(
-        self.fInfo,
-        self.fLocation.path.UTF8String,
-        trackerInfo,
-        self.fTrackers.count,
-        nullptr,
-        0,
-        self.fCommentView.string.UTF8String,
-        self.fPrivateCheck.state == NSControlStateValueOn,
-        NO,
-        self.fSource.stringValue.UTF8String);
-    tr_free(trackerInfo);
```

Comment:
@ckerr Are we moving current code from `tr_new`/`tr_malloc`/`tr_free` in general?

Reply:
Yes, absolutely.